rosdep/python.yaml

```diff
@@ -701,6 +701,14 @@ python-falcon:
   fedora: [python-falcon]
   ubuntu: [python-falcon]
 python-fcn-pip:
+  debian:
+    pip:
+      depends: [libhdf5-dev, liblapack-dev]
+      packages: [fcn]
+  fedora:
+    pip:
+      depends: [hdf5-devel, lapack-devel]
```

Comment:
+1

For mac, we does not have installation method for `liblapack-devel` (required for `numpy`), so we need to install requirements via `gfortran`.